scripts/deploy.sh

```diff
@@ -1,30 +1,28 @@
 #!/bin/bash
 set -e
 
-source scripts/travis_env.sh
-
 ARCHS=( armv7 x86 arm64_v8a x86_64 )
 
 COMMENT="Uploaded test apks for\n"
 for ARCH in "${ARCHS[@]}"
 do
-  COMMENT=${COMMENT}" - [**${ARCH}**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-${UPLOAD_ARTIFACT_ID}-${TRAVIS_COMMIT}-${ARCH}.apk)\n"
+  COMMENT=${COMMENT}" - [**${ARCH}**](https://download.opengis.ch/qfield/ci-builds/qfield-dev-${CI_UPLOAD_ARTIFACT_ID}-${CI_COMMIT}-${ARCH}.apk)\n"
```

Comment:
@suricactus for reference: the upload/download to/from download.opengis.ch will no longer be required with gh workflows as we can preserve artifacts directly inside the workflows, so this script should no longer be used (or shortened a lot) for gh workflows.